rust/driver/datafusion/Cargo.toml

```diff
@@ -32,17 +32,11 @@ categories.workspace = true
 
 [dependencies]
 adbc_core.workspace = true
-arrow-array = "55.2.0"
-arrow-buffer = "55.2.0"
-arrow-schema = "55.2.0"
```

Comment:
Are we now back to requiring that the Datafusion driver and the ADBC code libraries use the same Arrow version?